docs/resources/Channel.md

```diff
@@ -349,6 +351,7 @@ Update a channels settings. Requires the 'MANAGE_CHANNELS' permission for the gu
 | name | string | 2-100 character channel name | All |
 | position | integer | the position of the channel in the left-hand listing | All |
 | topic | string | 0-1024 character channel topic | Text |
+| nsfw | bool | if the channel is nsfw | Text |
```

Comment:
This actually can be set as true or false on categories and voice channels as well, but it doesn't have any effect. So this is correct, just FYI 👍